src/datalad_installer.py

```diff
@@ -984,11 +984,41 @@ class NeurodebianComponent(Component):
         ],
     )
 
+    KEY_FINGERPRINT = "0xA5D32F012649A5A9"
+    DOWNLOAD_SERVER = "us-nh"
+
     def provide(self, extra_args: Optional[List[str]] = None, **kwargs: Any) -> None:
         log.info("Installing & configuring NeuroDebian")
         log.info("Extra args: %s", extra_args)
         if kwargs:
             log.warning("Ignoring extra component arguments: %r", kwargs)
+        release = readcmd("lsb_release", "-cs").strip()
```

Comment:
lsb_release might not be available.  e.g. try in a base ubuntu:focal docker image. It would need to have `lsb-release` package installed.